cocoapods_installer_test.go

```diff
@@ -0,0 +1,150 @@
+package main
+
+import (
+	"strings"
+	"testing"
+
+	"bitrise-steplib/steps-cocoapods-install/mocks"
+
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_GivenCocoapodsInstaller_WhenArgsGiven_ThenRunsExpectedCommand(t *testing.T) {
+	type args struct {
+		podArg     []string
+		podCmd     string
+		podfileDir string
+		verbose    bool
+	}
+	tests := []struct {
+		name    string
+		args    args
+		wantErr bool
```

Comment:
`wantError` is always false in these tests. Do we need it?

Reply:
Not really, removed in https://github.com/bitrise-steplib/steps-cocoapods-install/pull/90/commits/31617e6585d98d3ec2d74690211c4d9d3d2c7ae7